client/src/components/IconTextField.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Box, Grid, InputAdornment, TextField } from "@mui/material";
+
+const IconTextField = ({ icon, placeholder, value, setValue }) => {
+  return (
+    <Grid item sx={{ backgroundColor: "white", borderRadius: "20rem" }}>
+      <TextField
+        hiddenLabel
+        fullWidth
+        placeholder={placeholder}
+        value={value}
+        onChange={(e) => setValue(e.target.value)}
+        InputProps={{
+          startAdornment: (
+            <Box padding="0.3rem">
```

Comment:
Is the extra box necessary or can we just apply the padding to the following adornment component? Is the sx prop available as well?